components/atom/videoPlayer/README.md

```diff
@@ -0,0 +1,34 @@
+# AtomVideoPlayer
+
+> Description
```

Comment:
Description?

Reply:
Thank you for the feedback.

This is still WIP and we'll take care of it during the following days.

Reply:
yeap. its a TODO for those who code it